src/Sylius/Bundle/ThemeBundle/Asset/Installer/OutputAwareAssetsInstaller.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Asset\Installer;
+
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class OutputAwareAssetsInstaller extends AssetsInstaller implements OutputAwareAssetsInstallerInterface
+{
+    /**
+     * @var OutputInterface
+     */
+    protected $output;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setOutput(OutputInterface $output)
+    {
+        $this->output = $output;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function installAssets($targetDir, $symlinkMask)
+    {
+        if ($this->hasOutput()) {
+            if (self::HARD_COPY !== $symlinkMask) {
+                $this->output->writeln('Trying to install assets as <comment>symbolic links</comment>.');
+            } else {
+                $this->output->writeln('Installing assets as <comment>hard copies</comment>.');
+            }
+        }
+
+        return parent::installAssets($targetDir, $symlinkMask);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function installBundleAssets(BundleInterface $bundle, $targetDir, $symlinkMask)
+    {
+        $sources = $this->findAssetsPathsForBundle($bundle);
+
+        if (0 === count($sources)) {
+            $effectiveSymlinkMask = $symlinkMask;
+        } else {
```

Comment:
Instead of indenting the whole method with else, we can make use of early return

```php
if (empty($sources)) {
    return $symlinkMask;
}
... // go on with the logic
```